Gruntfile.js

```diff
@@ -17,7 +17,7 @@ module.exports = function(grunt) {
   		dynamic: {
   			files: [{
   				expand: true,
-  				cwd: 'images/',
+  				cwd: 'images/public/',
```

Comment:
`cwd` should be `images/`, but `dest` should be outside of `images` directory - e.g. `public/`